test/glimesh/chat_test.exs

```diff
@@ -25,6 +25,21 @@ defmodule Glimesh.ChatTest do
       chat_message
     end
 
+    def multiple_chat_message_fixture(attrs \\ %{num_of_messages: 10}) do
+      channel = channel_fixture()
+      user = user_fixture()
+      num_of_messages = attrs.num_of_messages - 2
+
+      # Pretty janky way of creating multiple chat messages but it works
+      Enum.each(0..num_of_messages, fn _i -> Chat.create_chat_message(user, channel, attrs |> Enum.into(@valid_attrs)) end)
```

Comment:
You could remove line 31 entirely if you did

```suggestion
      created_messages = Enum.each(1..num_of_messages, fn _i -> Chat.create_chat_message(user, channel, attrs |> Enum.into(@valid_attrs)) end)
      
      {:ok, chat_message} = tl(created_messages)
```

Reply:
Unfortunately that wouldn't work as Enum.each just returns a :ok when it is finished. I'm using that solely to loop n times to create a chat message. Unless there is a function that takes the results from the inner function and puts it into a list.

Reply:
Oops, make it an Enum.map